### ISSUE TITLE
When implementing external endpoint reporters and app/web.config contains "Metrics.HttpListener.HttpUriPrefix", registration is attempted more than once and fails silently.

### ISSUE THREAD

**Issue by vic10us** [Recognos/Metrics.NET#38]
When I try to register (just an example using Metrics.NET.Prometheus)

```
            // this should not be necessary either...
            var metricsEndpoint = AppSettings["Metrics.HttpListener.HttpUriPrefix"] ?? "http://localhost:1234/someservice/";

            Metric.Config
                .WithHttpEndpoint(metricsEndpoint,
                    config =>
                        config.WithPrometheusEndpointReport())
                .WithSystemCounters()
                .WithAllCounters();
```

if the app.config contains Metrics.HttpListener.HttpUriPrefix key it will try to register the endpoint twice. Once for the base implementation and again when it attempts to add the report endpoint. Since the base endpoint is already registered it simply returns without error.

**Comment by PaulParau** [comment]
Hi, I changed it in the latest version ([v0.4.6-pre](https://www.nuget.org/packages/Metrics.NET/0.4.6-pre)): an exception is now thrown upon startup if an http endpoint is configured twice.

**Comment by PaulParau** [comment]
Fix released in the latest stable version [v0.4.7](https://www.nuget.org/packages/Metrics.NET/0.4.7)
